pkg/publish/daemon.go

```diff
@@ -43,8 +44,12 @@ func NewDaemon(namer Namer, tags []string) Interface {
 	return &demon{namer, tags}
 }
 
+var defaultClient = func() (daemon.Client, error) {
+	return client.NewClientWithOpts(client.FromEnv)
```

Comment:
I might have this be `func getOpts(ctx) { return WithContext(ctx) }` and override it to include `daemon.WithClient` for tests, so we don't have to import the docker client just to do what `daemon` does by default?

Reply:
Dang that's nice.